frontend/app/(dashboard)/updates/company/Edit.tsx

```
@@ -1,225 +1,525 @@
-    if (navigatedFromNewPreview) {
-      sessionStorage.removeItem("navigated-from-new-preview");
+    if (!id && selectedRecipients.length === 0) {
+      setSelectedRecipients(["administrators"]);
     }
-  }, []);
+  }, [id]);
+
+  // Close dropdown when clicking outside
+  useEffect(() => {
+    const handleClickOutside = (e: MouseEvent) => {
+      if (dropdownOpen && !(e.target as Element).closest(".recipient-selector")) {
```

ESLint

[error] Do not use any type assertions.

(@typescript-eslint/consistent-type-assertions)